### ISSUE TITLE
[Bug] Change Accessibility focus in Android

### ISSUE THREAD

**Issue by Poli97** [xamarin/Xamarin.Forms#9692]
### Description
Not able to change the accessibility focus on a certain element after a certain event (like a button press).
Asked also on [Community](https://forums.xamarin.com/discussion/178244/android-change-accessibility-focus)

### Steps to Reproduce

1. Create an Android project
2. Insert some random elements (like 3 buttons)
3. Handle the Click of one of the buttons
4. In the Click handler try to change the accessibility focus on one of the other buttons
5. Test the app on a physical device with TalkBack ON 

### Expected Behavior
When you tap a certain button the screen reader should change the focus to the wanted button

### Actual Behavior
Using the expected command, `myview.SendAccessibilityEvent(EventTypes.ViewAccessibilityFocused)` the screen reader only read the accessibility name of `myview` without actually setting the focus on it, that remain on the  pressed button.
Using some other commands, like `myview.SendAccessibilityEvent(EventTypes.ViewFocused)` or `b3.SendAccessibilityEvent(EventTypes.WindowsChanged)` nothing happen.
Using some other command like `myview.SendAccessibilityEventUnchecked(AccessibilityEvent.WindowsChangeAccessibilityFocused)` (that should actually be the right one that work in Android Studio), it give the error `Argument 1: cannot convert from 'Android.Views.Accessibility.WindowsChange' to 'Android.Views.Accessibility.AccessibilityEvent`

###Other information
In Android Studio, using Java, to set the accessibility focus on a certain `myview` you have to do `myview.sendAccessibilityEvent(AccessibilityEvent.WINDOWS_CHANGE_ACCESSIBILITY_FOCUSED)` and it works fine. In Xamarin, using C#, everything that I tried doesn't work properly.

### Reproduction Link

[Git repository example](https://github.com/Poli97/AndroidChangeFocus)

**Comment by jsuarezruiz** [comment]
Seems to be an issue related with Xamarin.Android. @samhouts can we move this issue to https://github.com/xamarin/xamarin-android/issues
